.github/workflows/python_hosted_runner_test.yml

```diff
@@ -152,12 +167,12 @@ jobs:
         if: always()
         uses: actions/upload-artifact@v4
         with:
-          name: test-results-${{ matrix.modules }}--${{ matrix.java }}-${{ inputs.hadoop }}-hive2.3-${{ env.PYTHON_TO_TEST }}
+          name: test-results-${{ inputs.os }}-${{ matrix.modules }}--${{ matrix.java }}-${{ inputs.hadoop }}-hive2.3-${{ env.PYTHON_TO_TEST }}
```

Comment:
This change is made to facilitate the distinction of the results uploaded by the `pyspark` task in `build_and_test` during manual testing.